docs/start-stop.rst

```diff
@@ -117,3 +117,32 @@ with a README documenting the installation instructions, is available
 within the source repository at ``util/scripts/lsfcsm``.
 
 Support for the SLURM resource manager is under development.
+
+-----------------------------------------------
+  Stage-in and Stage-out Manifest File Format
+-----------------------------------------------
+
+The --stage-in and --stage-out options for unify (start and terminate,
+respectively) require a manifest file to specifiy the files to move.
+The manifest file is a one-for-one list of files to be copied from and copied
```

Comment:
Replace whole paragraph with: "The manifest file contains one or more file copy requests. Each line in the manifest corresponds to one file copy request, and contains both the source and destination file paths. Currently, directory copies are not supported."

Reply:
Implementing all these changes then I'll push again.